modules/salt/pages/formula-tftpd.adoc

```diff
@@ -1,7 +1,13 @@
 [[tftpd-formula]]
 = TFTPd Formula
 
-The TFTPd formula is used to configure the TFTP service on the branch server.
+The TFTPd formula is used to configure the TFTP service on the {productname} Proxy.
+
```

Comment:
"TFTPd formula" vs. "`tftpd` formula" spelling.  I'd uppercase it erverywhere (TFTPD) or use two words (TFTP daemon). Software to be changed accordingly, IMO.